packages/components/src/DataList/components/DataListEmptyState/DataListEmptyState.tsx

```diff
@@ -20,12 +21,41 @@ export function InternalDataListEmptyState() {
     useContext(DataListContext);
   const { message, action } = getMessages();
 
-  return (
-    <div className={styles.emptyStateWrapper}>
-      <Text align="center">{message}</Text>
-      {renderButton(action)}
-    </div>
-  );
+  if (filtered) {
+    return (
+      <div className={styles.emptyStateWrapper}>
+        <Text align="center">{message}</Text>
+        {renderButton(action)}
+      </div>
+    );
+  } else {
+    return (
+      <div className={styles.emptyStateWrapper}>
```

Comment:
As noted elsewhere, this could probably be moved out to its own component

Reply:
Alternatively, could these actually live in the columns of the `DataList.Layout` so you consistently get a more representative empty state?